waiter/src/waiter/settings.clj

```diff
@@ -179,6 +179,12 @@
                                    (s/required-key :token-defaults) {(s/required-key "fallback-period-secs") schema/non-negative-int
                                                                      (s/required-key "https-redirect") s/Bool
                                                                      (s/required-key "stale-timeout-mins") schema/non-negative-int}}
+   (s/required-key :watch-config) {(s/required-key :token-watch-maintainer) {(s/required-key :channels-update-chan-buffer-size)
```

Comment:
Let's keep the values in the same line as the keys.

Reply:
Maybe rename `token-watch-maintainer` to `tokens`, it is already nested inside `watch-config`?